lib/_stream_writable.js

```diff
@@ -307,7 +303,12 @@ function writeOrBuffer(stream, state, chunk, encoding, cb) {
 
   if (state.writing || state.corked) {
     var last = state.lastBufferedRequest;
-    state.lastBufferedRequest = new WriteReq(chunk, encoding, cb);
+    state.lastBufferedRequest = {
+      chunk: chunk,
+      encoding: encoding,
+      callback: cb,
+      next: null
+    };
```

Comment:
I thought about that, but wasn't sure if that would cause compatibility problems for `readable-stream` users or if the babel transforms took care of that or what...